x-pack/plugins/security_solution/common/api/entity_analytics/entity_store/engine/stop.schema.yaml

```
@@ -0,0 +1,30 @@
+openapi: 3.0.0
+
+info:
+  title: Stop the entity store engine
+  version: '1'
```

I believe that public APIs should use a specific date for version.
```suggestion
  version: '2023-10-31'
```